config/conf.json

```diff
@@ -84,7 +84,7 @@
         },
         {
             "name": "terra-jupyter-gatk",
-            "base_label": "New Default (released on January 14)",
+            "base_label": "Default",
```

Comment:
It seems like changes in this file didn't take in the generated json.. will investigate why.

Reply:
hmm... If you generate the JSON after bumping the version, and then change the label and try to generate it again, it will see that the remote version matches your local and just use the remote doc. Could be what's happening here. 

TBH, it may be worth removing this branch of the if in `generate_docs`, since its not like we save too much time by checking the remote version here, and it can be annoying. All you have to do is delete this branch of the if. The original intent was that we didn't want changes in labelling/etc to be built by the CI before an explicit version bump
```
      if image_config["name"] in remote_versions and image_config["version"] == remote_versions[image_config["name"]]:
```
A

Reply:
Thanks.. yeah I came to the same conclusion and re-generated the docs. I think it looks good now, updated the PR description.